linode/manager
# M3-5918: Remove `babel` and `webpack` following use of `tsup`

## Description

- Remove unused dependencies and clean up code in the api-v4 and validation packages following #8484

## How to test

- Ensure `@linode/api-v4` and `@linode/validation` still build and work properly in local development and in our ci pipelines
